spec/registrant_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+  before(:each) do
+
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    # pry(main)> registrant_1 = Registrant.new('Bruce', 18, true )
+    # #=> #<Registrant:0x000000015c10bed8 @age=18, @license_data={:written=>false, :license=>false, :renewed=>false}, @name="Bruce", @permit=true>
+    
+    @registrant_2 = Registrant.new('Penny', 15 )
+    # pry(main)> registrant_2 = Registrant.new('Penny', 15 )
+    # #=> #<Registrant:0x000000015c0778c8 @age=15, @license_data={:written=>false, :license=>false, :renewed=>false}, @name="Penny", @permit=false>
+    
+  end
+
+  it "can initialize" do
+    expect(@registrant_1).to be_an_instance_of(Registrant)
+    expect(@registrant_2).to be_an_instance_of(Registrant)
+  end
+
+  it "has a name" do
+    expect(@registrant_1.name).to eq("Bruce")
+    # pry(main)> registrant_1.name
+    # #=> "Bruce"
+    
+    expect(@registrant_2.name).to eq("Penny")
+    # pry(main)> registrant_2.name
+    # #=> "Penny"
+  end
+  
+  it "has an age" do
+    expect(@registrant_1.age).to eq(18)
+    # pry(main)> registrant_1.age
+    # #=> 18
+    
+    expect(@registrant_2.age).to eq(15)
```

Comment:
For these attributes and the check if it's an instance of Registrant, really only need to test one instance unless there's any flow control going on.